include/aikido/robot/Hand.hpp

```diff
@@ -37,7 +37,8 @@ class Hand
   virtual void step(const std::chrono::system_clock::time_point& timepoint) = 0;
 
   /// Returns the metaskeleton corresponding to this hand.
-  virtual dart::dynamics::MetaSkeletonPtr getMetaSkeleton() const = 0;
+  virtual dart::dynamics::ConstMetaSkeletonPtr getMetaSkeleton() const = 0;
+  virtual dart::dynamics::MetaSkeletonPtr getMetaSkeleton() = 0;
```

Comment:
Nit: We need docstring for an individual function. The second function wouldn't have a description. :sob: